src/ostorlab/assets/ipv4.py

```diff
@@ -11,6 +11,10 @@
 class IPv4(asset.Asset):
     """IPv4 Address target asset."""
 
+    host: str
+    version: Optional[int] = 4
```

Comment:
Version can't be None, the typing in init was not correct!
```suggestion
    version: int = 4
```

Reply:
Done